datasets/NJ_Imagery.yaml

```
@@ -0,0 +1,17 @@
+Name: Digital Aerial Imagery Catalog 

```

Please add something indication this is from NJ, something like below
```suggestion
Name: New Jersey Statewide Digital Aerial Imagery Catalog 

```